docs/config.md

```diff
@@ -116,6 +116,18 @@ This maps directly to the [Puma bind
 flag](https://github.com/puma/puma#binding-tcp--sockets), and will support
 anything valid for that flag.
 
+## Environment Variables
```

Comment:
What do you think about moving this section to be a `###` subsection of `## Config File Location`? It feels to me like it might flow well after the discussion about customizing the config location. What do you think about phrasing it in terms of ERB parsing files that end with `.erb`, with perhaps a blurb about using it to embed environment variables in the config file?